examples/producer.go

```diff
@@ -24,20 +24,23 @@ import (
 )
 
 func main() {
-	SendMessage()
-}
+	cfg := &rocketmq.ProducerConfig{}
+	cfg.GroupID = "testGroup"
+	cfg.NameServer = "47.101.55.250:9876"
+	producer, err := rocketmq.NewProducer(cfg)
+	if err != nil {
+		fmt.Println("create Producer failed, error:", err)
+		return
+	}
 
-func SendMessage() {
-	producer := rocketmq.NewProducer(&rocketmq.ProducerConfig{GroupID: "testGroup", NameServer: "localhost:9876"})
 	producer.Start()
 	defer producer.Shutdown()
 
 	fmt.Printf("Producer: %s started... \n", producer)
 	for i := 0; i < 100; i++ {
 		msg := fmt.Sprintf("Hello RocketMQ-%d", i)
-		result := producer.SendMessageSync(&rocketmq.Message{Topic: "test", Body: msg})
+		result := producer.SendMessageSync(&rocketmq.Message{Topic: "wwf1", Body: msg})
```

Comment:
It is better to change the topic name back to "Test".

Reply:
sorry, i forgot change it back.